README.md

```diff
@@ -337,3 +337,14 @@ Any valid tag can be run from this command line not just the feature tag name.
 Currently, we have a BaseSteps class which statically loads some properties at startup. The idiomatic Cucumber standard is to have an object called a 'World' which maintains such state. Such a class has been created for this purpose, called, naturally, World. Further, we are leveraging cucumber-picocontainer to inject instances of this World class into steps. Why Picocontainer? Cucumber-JVM supports a number of dependency injection containers (Spring, Guice, Picocontainer, something called 'Weld') but Picocontainer support is the most mature, and the lightest touch in terms of transitive dependencies.
 
 Ultimately, the intent is that anything that is not a Cucumber step definition will not live in any steps classes. This may seem a little unusual to a Java developer but a quirk of Cucumber-JVM is that you cannot extend any class which has @Before or @After hooks, so should we wish to provide such hooks, a BaseSteps class is not the way to achieve that. (Note that those annotations are from the package cucumber.api.java not org.junit).
+
+# Debugging Cucumber tests in IntelliJ
+
+1. Open the feature file of the scenario that you want to debug and locate the scenario
+2. Right click on the scenario and select "Debug: 'Scenario: Test...'"
```

Comment:
Instead of clicking Debug and waiting for the tests to fail you can just click Create X which also opens the window to customise parameters

![image](https://user-images.githubusercontent.com/15050019/32836696-9c3b5cac-ca02-11e7-842a-2d3d83727890.png)

Reply:
Done